paimon-web-server/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM eclipse-temurin:8-jre
+
+ENV DOCKER true
+ENV TZ Asia/Shanghai
+ENV PAIMON_UI_HOME /opt/paimon-webui
+
+WORKDIR $PAIMON_UI_HOME
+
+ADD ./target/paimon-web-server $PAIMON_UI_HOME
```

Comment:
Does this need to regard `./target/paimon-web-server` as `PAIMON_UI_HOME`?

Reply:
His statement means that when the Docker command is executed, the contents of the ./target/paimon-web-server directory will be copied into the PAIMON_UI_HOME directory within the container. This Dockerfile is designed such that during the Maven build process, the image can be pushed to DockerHub.